themes/index.js

```diff
@@ -268,6 +268,24 @@ const themes = {
     icon_color: "ff1aff",
     text_color: "8080ff",
     bg_color: "141439",
+  },
+  "rose-pine":{
+    title_color: "9CCFD8",
+    icon_color: "C3A6E6",
+    text_color: "FAF4ED",
+    bg_color: "1f1d29",
+  },
+  "rose-pine-moon":{
+    title_color: "9CCFD8",
+    icon_color: "C4A7E7",
+    text_color: "FAF4ED",
+    bg_color: "232136",
+  },
```

Comment:
I just tested this, and i think we don't need this moon theme... because there is not enough elements in our cards to see any significant changes by adding this theme too. they are very similar. 

We can just keep the `original` & the `dawn`

Reply:
Done